pkg/minishift/registration/registration.go

```diff
@@ -62,18 +62,18 @@ func Register(name string, r *RegisteredRegistrator) {
 	registrators[name] = r
 }
 
-func DetectRegistrator(c provision.SSHCommander) (Registrator, error) {
+func DetectRegistrator(c provision.SSHCommander) (Registrator, bool, error) {
 	return detector.DetectRegistrator(c)
 }
 
-func (detector StandardRegistrator) DetectRegistrator(c provision.SSHCommander) (Registrator, error) {
+func (detector StandardRegistrator) DetectRegistrator(c provision.SSHCommander) (Registrator, bool, error) {
 	osReleaseOut, err := c.SSHCommand("sudo cat /etc/os-release")
 	if err != nil {
-		return nil, err
+		return nil, false, err
 	}
 	osReleaseInfo, err := provision.NewOsRelease([]byte(osReleaseOut))
 	if err != nil {
-		return nil, fmt.Errorf("Error parsing /etc/os-release file: %s", err)
+		return nil, false, fmt.Errorf("Error parsing /etc/os-release file: %s", err)
 	}
 
 	log.Info("Detecting Host VM is managed by registration manager...")
```

Comment:
I don't think we need this. I think this might be a bit confusing for most of the community folks where registration is nothing they know about. I suggest to change this to some sort of debug trace. Also it should be something like:"Detecting whether virtual machine is managed by a registration manager..."